go-controller/pkg/ovn/controller/services/services_controller_test.go

```diff
@@ -627,7 +528,174 @@ func TestSyncServices(t *testing.T) {
 				t.Errorf("syncServices error: %v", err)
 			}
 
-			g.Eventually(controller.nbClient).Should(libovsdbtest.HaveData(tt.expectedDb))
+			g.Expect(controller.nbClient).To(libovsdbtest.HaveData(tt.expectedDb))
+
+			if tt.nodeToDelete != nil {
+				controller.nodeTracker.removeNode(tt.nodeToDelete.name)
+
+				// we need to extract the deleted load balancer UUID, because
+				// of a test library limitation: it does not delete the weak
+				// references to the load balancers on the logical switches /
+				// logical routers.
+				nodeLoadBalancerUUID, err := extractLoadBalancerRealUUID(
+					controller.nbClient,
+					nodeSwitchRouterLoadBalancerName(nodeA, ns, serviceName))
+				g.Expect(err).NotTo(gomega.HaveOccurred())
+				g.Expect(nodeLoadBalancerUUID).NotTo(gomega.BeEmpty())
+
+				g.Expect(controller.syncService(namespacedServiceName(ns, serviceName))).To(gomega.Succeed())
+
+				// here, we need to mutate the original expected data with the
```

Comment:
I don't think that `mutate` is the better word to use here, because it can be confused with the real ovsdb Mutate operation. `Patch` probably a better choice.

Reply:
Done.